core/src/main/java/org/apache/accumulo/core/fate/FateExecutor.java

```diff
@@ -201,6 +209,12 @@ protected Set<Fate.FateOperation> getFateOps() {
     return fateOps;
   }
 
+  public void printInfo() {
+    synchronized (runningTxRunners) {
+      runningTxRunners.forEach(r -> r.printInfo());
+    }
+  }
+
```

Comment:
#5824 does a similar thing, but adds the info to the existing toString. I think the toString additions are a bit cleaner